### PR TITLE
Removed white space for select queries by formatting to single line

### PR DESCRIPTION
**Pull request by LukaszJaro**
I removed the white space for any select query by formatting it to one line. This keeps it consistent with the other select queries which are also one line.

Trac ticket: https://core.trac.wordpress.org/ticket/56841

---
**This Pull Request is for code review only. Please keep all other discussion in the Trac ticket. Do not merge this Pull Request. See [GitHub Pull Requests for Code Review](https://make.wordpress.org/core/handbook/contribute/git/github-pull-requests-for-code-review/) in the Core Handbook for more details.**

### CODE REVIEW

**Comment by LukaszJaro** [comment]
> The following accounts have interacted with this PR and/or linked issues. I will continue to update these lists as activity occurs. You can also manually ask me to refresh this list by adding the `props-bot` label.
> 
> ## Unlinked Accounts
> The following contributors have not linked their GitHub and WordPress.org accounts: @LukaszJaro.
> 
> Contributors, please [read how to link your accounts](https://make.wordpress.org/core/2020/03/19/associating-github-accounts-with-wordpress-org-profiles/) to ensure your work is properly credited in WordPress releases.
> 
> **To understand the WordPress project's expectations around crediting contributors, please [review the Contributor Attribution page in the Core Handbook](https://make.wordpress.org/core/handbook/best-practices/contributor-attribution-props/).**

Refresh Please!

File: src/wp-includes/class-wp-comment-query.php [hunk 1]
```diff
@@ -964,14 +964,12 @@ protected function get_comment_ids() {
...
-			{$this->sql_clauses['select']}
-			{$this->sql_clauses['from']}
-			{$where}
-			{$this->sql_clauses['groupby']}
-			{$this->sql_clauses['orderby']}
-			{$this->sql_clauses['limits']}
-		";
+		$this->request =
+			"{$this->sql_clauses['select']}
+			 {$this->sql_clauses['from']} {$where}
```
Comment (ironprogrammer) [inline src/wp-includes/class-wp-comment-query.php]:
```suggestion
			 {$this->sql_clauses['from']}
			 {$where}
```
Having `WHERE` on a separate line seems more easily readable.

File: src/wp-includes/class-wp-network-query.php [hunk 2]
```diff
@@ -481,14 +481,12 @@ protected function get_network_ids() {
...
-			{$this->sql_clauses['select']}
-			{$this->sql_clauses['from']}
-			{$where}
-			{$this->sql_clauses['groupby']}
-			{$this->sql_clauses['orderby']}
-			{$this->sql_clauses['limits']}
-		";
+		$this->request =
+			"{$this->sql_clauses['select']}
+			 {$this->sql_clauses['from']} {$where}
```
Comment (ironprogrammer) [inline src/wp-includes/class-wp-network-query.php]:
```suggestion
			 {$this->sql_clauses['from']}
			 {$where}
```
Same here.

File: src/wp-includes/class-wp-site-query.php [hunk 3]
```diff
@@ -695,14 +695,12 @@ protected function get_site_ids() {
...
-			{$this->sql_clauses['select']}
-			{$this->sql_clauses['from']}
-			{$where}
-			{$this->sql_clauses['groupby']}
-			{$this->sql_clauses['orderby']}
-			{$this->sql_clauses['limits']}
-		";
+		$this->request =
+			"{$this->sql_clauses['select']}
+			 {$this->sql_clauses['from']} {$where}
```
Comment (ironprogrammer) [inline src/wp-includes/class-wp-site-query.php]:
```suggestion
			 {$this->sql_clauses['from']}
			 {$where}
```
Same here.

**Review by ironprogrammer — COMMENTED** [review]
Looks good, @LukaszJaro! I've left some suggestions below that maintain the readability intent from [r52973](https://core.trac.wordpress.org/changeset/52973).

**Comment by LukaszJaro** [comment]
> Looks good, @LukaszJaro! I've left some suggestions below that maintain the readability intent from [r52973](https://core.trac.wordpress.org/changeset/52973).

Thanks @ironprogrammer for getting back, I did not get this email notification unfortunately(email sync issue), I just committed the changes.

**Review by ironprogrammer — APPROVED** [review]
LGTM!

**Comment by ironprogrammer** [comment]
As [proposed](https://core.trac.wordpress.org/ticket/56841#comment:34) by @afercia, should each of the updated strings include something along the lines of:
```php
// Remove leading and trailing whitespace, but maintain readability.
```

**Review by swissspidy — APPROVED** [review]

**Comment by swissspidy** [comment]
Committed in https://core.trac.wordpress.org/changeset/57750

**Pull request closed without merging**